sdk/tools/azure-openrewrite/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -0,0 +1,255 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: com.azure.openrewrite.migrateToVNext
```

Comment:
Should give this a more descriptive name as this is very generic.

Reply:
@alzimmermsft This won't be the final name. It's just to get it into main.